### PR TITLE
update changelogs

### PR DESCRIPTION
**Pull request by rohitthakur2590**
Signed-off-by: Rohit Thakur <rohitthakur2590@outlook.com>

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

NA
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: changelogs/changelog.yaml [hunk 1]
```diff
@@ -124,3 +124,19 @@ releases:
...
+  1.0.1:
+    changes:
+      bugfixes:
+      - Make `src`, `backup` and `backup_options` in junos_config work when module
+        alias is used (https://github.com/ansible-collections/junipernetworks.junos/pull/83).
+      - Update docs after sanity fixes to modules.
+      minor_changes:
+      - Add changelogs for 1.0.0 release (https://github.com/ansible-collections/junipernetworks.junos/pull/70)
+      release_summary:
+      - Rereleased 1.0.0 with updated changelog.
```
Comment (Qalthos) [inline changelogs/changelog.yaml:136]: If we still haven't released a 1.0.1. I don't think that's correct.

File: changelogs/changelog.yaml [hunk 2]
```diff
@@ -124,3 +124,16 @@ releases:
...
         namespace: null
     release_date: '2020-06-23'
+  1.0.1:
+    changes:
+      bugfixes:
+      - Make `src`, `backup` and `backup_options` in junos_config work when module
+        alias is used (https://github.com/ansible-collections/junipernetworks.junos/pull/83).
+      - Update docs after sanity fixes to modules.
+      minor_changes:
+      - Add changelogs for 1.0.0 release (https://github.com/ansible-collections/junipernetworks.junos/pull/70)
```
Comment (NilashishC) [inline changelogs/changelog.yaml:134]: The entry is not required. This got generated because the changelog PR for 1.0.0 had a changelog entry

**Comment by rohitthakur2590** [comment]
recheck

File: changelogs/changelog.yaml (hunk 2 above)
Comment (Qalthos) [reply changelogs/changelog.yaml:134]: And the line in CHANGELOG, and the fragment `70-add-change-log-1.0.0.yaml`.

**Pull request merged**